docs/site/development/cldr-big-red-switch.md

```diff
@@ -12,6 +12,43 @@ The checklist can be filtered to see open tasks by going to Data > Change view a
 
 TODO: List all major types of tasks with their definitions and link to pages with descriptions.
 
+## Jira filters for release management and issue triage
+
+### Ticket Triage
+
+- [@New, by date](https://unicode-org.atlassian.net/issues/?filter=10033) - All untriaged issues sorted by date created.
+- [@New, by fix version](https://unicode-org.atlassian.net/issues/?filter=10801) - All untriaged issues sorted by fix version when available.
+- [@New + Agenda, by date](https://unicode-org.atlassian.net/issues/?filter=10802)
+- [@Agenda](https://unicode-org.atlassian.net/issues/?filter=10158) - All issues tagged for TC discussion.
+- [CLDR + ICU helpwanted](https://unicode-org.atlassian.net/issues/?filter=10202) - Tickets tagged as 'Help Wanted' by the CLDR or ICU Technical Committees
+
+### Ticket Reviewing
+
+- [@Pending Reviews](https://unicode-org.atlassian.net/issues/?filter=10062) - All CLDR tickets currently in review.
+- [@My Reviews](https://unicode-org.atlassian.net/issues/?filter=10179) - CLDR tickets in review assigned to the viewer.
```

Comment:
"the viewer" was unclear to me until I clicked the link and figured it out. I don't have a great idea about rephrasing, maybe "assigned to oneself"?